test/unit/helpers/admin/publications_table_helper_test.rb

```diff
@@ -0,0 +1,185 @@
+require "test_helper"
+
+class PublicationsTableHelperTest < ActionView::TestCase
+  include PublicationsTableHelper
+
+  context "#edition_number" do
+    should "return the edition number for an edition that is not archived or published " do
+      edition = FactoryBot.create(
+        :edition,
+        state: "draft",
+        version_number: 1,
+      )
+
+      assert_equal "1", edition_number(edition)
+    end
+
+    should "return the edition number text for a published edition where a newer edition has been created" do
+      artefact = FactoryBot.create(:artefact)
+      published_edition = FactoryBot.create(
+        :edition,
+        panopticon_id: artefact.id,
+        state: "published",
+        version_number: 1,
+        sibling_in_progress: 2,
+      )
+      FactoryBot.create(
+        :edition,
+        panopticon_id: artefact.id,
+        state: "draft",
+        version_number: 2,
+      )
+
+      edition_number_text = edition_number(published_edition)
+
+      assert_match "1 - ", edition_number_text
+      assert_match "#2 in draft", edition_number_text
+    end
+
+    should "return the edition number text for an archived edition where a newer edition has been created" do
+      artefact = FactoryBot.create(:artefact)
+      archived_edition = FactoryBot.create(
+        :edition,
+        panopticon_id: artefact.id,
+        state: "archived",
+        version_number: 2,
+        sibling_in_progress: 3,
+      )
+      FactoryBot.create(
+        :edition,
+        panopticon_id: artefact.id,
+        state: "in_review",
+        review_requested_at: "2024-07-12 11:25:35.297 UTC",
+        version_number: 3,
+      )
+
+      edition_number_text = edition_number(archived_edition)
+
+      assert_match "2 - ", edition_number_text
+      assert_match "#3 in in review", edition_number_text
+    end
+  end
+
+  context "#important_note" do
+    should "return the important note text for an edition" do
+      user = FactoryBot.create(:user, name: "Keir Starmer")
+      edition = FactoryBot.create(:edition)
+      user.record_note(edition, "This is an important note", Action::IMPORTANT_NOTE)
+
+      assert_equal "This is an important note", important_note(edition)
+    end
+  end
+
+  context "#awaiting_review" do
+    should "return the correct text for the 'Awaiting review' field for an edition in review" do
+      today = Date.parse("2024-08-02")
+
+      Timecop.freeze(today) do
+        edition = FactoryBot.create(
+          :edition,
+          state: "in_review",
+          review_requested_at: "2024-07-12",
+        )
+
+        assert_equal "21 days", awaiting_review(edition)
+      end
+    end
+  end
+
+  context "#scheduled" do
+    should "return the correct text for the 'Scheduled' field for an edition in 'Scheduled for publishing'" do
```

Comment:
I think we can slightly simplify the test name. For example:
```suggestion
    should "return the correct text for an edition that is scheduled for publishing" do
```